tests/math/test_polyfit.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test for custom polyfit"""
+# pylint: disable=import-outside-toplevel
+
+import pytest
+import pennylane as qml
+
```

Comment:
Wasnt sure where to put these, so just created a new file. Happy to move it somewhere more appropriate